blog-home.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog-home
+template-title: Blog Home
+template-description: Card-based landing page for a blog with media cards, tags, and secondary navigation
+title: Blog
+description: Latest updates on how agencies are serving the American people
+source-domain: usa.gov
+source-url: https://blog.usa.gov/
+blog-posts:
+    - title: "Making it Easier for People to find Federal Agencies"
+      author: "Joanne McGovern"
+      date: "June 08, 2021"
+      image: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/blog-2.jpg
+      labels:
+        - name: User Experience
+        - name: Accessibility
+        - name: Content
+        - name: Design
+        - name: Development
+        - name: Multilingual
+    - title: "Partner Spotlight: Informing the Public About the 2020 Census"
+      author: "Claire Loxsom"
+      date: "May 04, 2021"
+      image: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/blog-1.jpg
+      labels:
+        - name: Multilingual
+        - name: Content
+        - name: Performance
```

Comment:
"Performance and Analytics" is actually one tag. This is good to try so we can ensure longer tag labels respond well to different screen sizes.